test/k8sT/Policies.go

```diff
@@ -949,15 +949,15 @@ var _ = Describe("K8sPolicyTest", func() {
 						namespaceForTest,
 						"vagrant-cache.ci.cilium.io.",
 					)
-					curlCmd = helpers.CurlFail(resource)
+					curlCmd = helpers.CurlFail("--retry 5 " + resource)
```

Comment:
I'm not sure I really like this. Most of these are for internal destinations. We shouldn't depend on retries there. Can you pass in a flag instead which indicates whether to retry or not? We should only retry for the `worldTarget` case.

Reply:
Makes sense, thanks!